bpf/lib/nat.h

```diff
@@ -477,6 +478,65 @@ static __always_inline int snat_v4_icmp_rewrite_embedded(struct __ctx_buff *ctx,
 	return 0;
 }
 
+static __always_inline int
+snat_v4_icmp_rewrite_egress_embedded(struct __ctx_buff *ctx,
```

Comment:
Can we add at least a small comment to explain what this function does and why? From calling context I can infer what it does, but that requires me to go to a call site to understand it.